packages/ckeditor5-table/lang/contexts.json

```diff
@@ -64,9 +64,9 @@
 	"Insert a new table row (when in the last cell of a table)": "Keystroke description for assistive technologies: keystroke for inserting a new table row.",
 	"Navigate through the table": "Keystroke description for assistive technologies: keystroke for navigating through the table.",
 	"Table": "The accessible label of the menu bar button that displays a user interface to insert a table into editor content.",
-	"Table layout": "The accessible label of the menu bar or dropdown button that displays a user interface to insert a table layout into editor content.",
-	"Table content": "The accessible label of the menu bar or dropdown button that displays a user interface to insert table content into editor.",
-	"Choose table type": "The accessible label of the table ballon button that displays a user interface to choose table type.",
-	"Table type": "The accessible label of the table balloon button shown when the button of properties plugin is not present.",
+	"Table layout": "The accessible label of the table layout insert UI element.",
+	"Table content": "The accessible label of the table content insert UI element.",
```

Comment:
Does it insert table content, or does it insert content (traditional) table?

Also seeing it again, we may consider rewording this to "table inserting UI element", @gorzelinski - what say you?

Reply:
Updated description in https://github.com/ckeditor/ckeditor5/pull/18135/commits/91b0ece5502547e7b2ac15eda9dcc8ae897a34e5

Reply:
"UI element" can mean both "button" and "a dropdown" that's why I suggested it to make the message shorter, no point in including both.

What I meant here was the difference between "table content" - the text and stuff, and "content table" - the type of table, as opposed to "table layout" or "layout table".